src/host/utf8ToWideCharParser.cpp

```diff
@@ -505,5 +508,5 @@ void Utf8ToWideCharParser::_Reset()
 {
     _currentState = _State::Ready;
     _bytesStored = 0;
-    _convertedWideChars.release();
+    delete _convertedWideChars.release();

```

Comment:
I would prefer `_convertedWideChars.reset(nullptr);` here instead of an explicit delete.